### PR TITLE
Create an option to enable vtkweb

### PR DESCRIPTION
**Pull request by jbeezley**
It defaults to off so nothing should blow up.

### CODE REVIEW

File: CMake/cdat_modules/vtk_external.cmake [hunk 1]
```diff
@@ -73,6 +73,12 @@ if(CDAT_BUILD_OFFSCREEN)
   )
 endif()
 
+if(CDAT_BUILD_VTKWEB)
```
Comment (aashish24) [inline CMake/cdat_modules/vtk_external.cmake]: How about CDAT_ENABLE_WEB? Or CDAT_BUILD_WEB?

Comment (jbeezley) [inline CMake/cdat_modules/vtk_external.cmake]: `CDAT_BUILD_*` is consistent with the other options.  To me, `VTKWEB` is more explicit unless there is something else that `WEB` will enable, but I don't have a strong opinion.

Comment (doutriaux1) [inline CMake/cdat_modules/vtk_external.cmake]: I like that: CDAT_BUILD_VTKWEB

Comment (aashish24) [inline CMake/cdat_modules/vtk_external.cmake]: @doutriaux1 we don't want to expose vtkweb. Web option in the future will enable more than just vtkWeb.

**Comment by aashish24** [comment]
LGTM :+1:

**Pull request merged**